src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,13 @@
-export const UserInfo = () => {};
+import { User } from '../../types';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  return (
+    <a href={`mailto:${user.email}`} className="UserInfo UserInfo__link">
```

Comment:
Ensure that the `user` prop always contains valid `name` and `email` properties. If either is missing or invalid, the link might not render as expected.